packages/start/runtime/devServer.js

```diff
@@ -15,7 +15,7 @@ export function createDevHandler(viteServer) {
     try {
       if (req.url === "/favicon.ico") return;
 
-      const entry = (await viteServer.ssrLoadModule(path.resolve("/src/entry-server"))).default;
```

Comment:
This former line is actually from Vite's SSR example, except the part for the `path.resolve`.
<img width="643" alt="Screen Shot 2022-01-30 at 9 45 06 PM" src="https://user-images.githubusercontent.com/4783372/151702313-79f3f05d-b55d-4a25-b357-e09d0c986d40.png">

What happens if we remove `path.resolve` instead?

Reply:
I added path resolve to handle both js and ts entry points